login.js

```diff
@@ -56,8 +56,11 @@ var enterData = function(authData) {
 				answerInput.value = authData['ERPIITKGP_answer3'];
 				sessionStorage.setItem("q3", "1");
 			}
-
-			document.forms[0].submit();
+			detect_network(function(on_campus_network) {
+				if (on_campus_network) {
+					document.forms[0].submit();
+				}
```

Comment:
Let's add an else here where if it's not on campus network, the send OTP button is clicked automatically.

Reply:
Done.